<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentDemandWidget</class>
 <widget class="QWidget" name="ExperimentDemandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,2">
   <item>
    <widget class="QTabWidget" name="tabWidgetDisplay">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidgetData">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabRunnningExperiments">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEditRunExperToken"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonLiveExper">
            <property name="text">
             <string> 观看</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewRunningExperiments"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHistoryExperiment">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditHisExperToken"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPlaybackExper">
            <property name="text">
             <string>回放</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarHistory">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPlaybackExperFlush">
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>60</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewHistoryExperiment"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
